### ISSUE TITLE
Not compatible with Collection2 v3.0.0

### ISSUE THREAD

**Issue by kevinkorte** [MeteorToys/allthings#108]
I love this package - actually consider it an essential. However, Aldeed's Collection2 went to v3.0.0 and `meteortoys:allthings` now fails. Here's the message:

`Errors prevented startup:`

   `While selecting package versions:`
   `error: Conflict: Constraint aldeed:collection2@2.3.2 is not satisfied by aldeed:collection2 3.0.0.`
   `Constraints on package "aldeed:collection2":`
   `* aldeed:collection2@3.0.0 <- top level`
   `* aldeed:collection2@2.3.2 <- msavin:mongol 4.0.1 <- meteortoys:allthings 4.0.0`

What I've found works as a temporary solution is to roll `aldeed:collection2@3.0.0` to `aldeed:collection2@2.3.2` and Meteor will start up normally.

**Comment by johnwils** [comment]
Also love this tool, I mainly use the 'mongol' sub package (the bottom left display box).

That at least is currently working with `aldeed:collection2@3.0.0` installed.

`meteor add msavin:mongol`
`meteor add session`

Add css rule (displayed transparent otherwise):
```
#Mongol {
  background-color: #000;
}
```

**Comment by charlesdeb** [comment]
@johnwils - there is something in my package dependency that prevents your solution from working for me. Although mongol installs OK, it's version 0.5.5 (which I think is pretty old), the other package it depends on (`dburles:mongo-collection-instances`) seems to clash with `hwillson:stub-collections` - which has resulted in some broken tests. I will do some more digging to see if I can find the clash.

I got to this page because trying to do `meteor add meteortoys:allthings` resulted in
```
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint msavin:mongol@4.0.0 is not satisfied by msavin:mongol 0.1.1.
Constraints on package "msavin:mongol":
* msavin:mongol@4.0.0 <- meteortoys:allthings 4.0.0
```
It took __ages__ for the package resolver to come back to me on this also. I wonder if it is connected to collection v3.0.0 since I had a bunch of hassle trying to upgrade a bunch of aldeed's packages to the latest versions.

**Comment by erisevaline** [comment]
@johnwils 
Where do you add the CSS rule ? I put it in the automatic generated main.css (which is empty when meteor create it).
I still see the tool in transparency after doing the CTRL + M shortcut.

**Comment by johnwils** [comment]
@charlesdeb hard to determine the clash without removing packages one by one. Here a link to a somewhat 'full' working example if it helps: https://github.com/johnwils/meteor-react-template

@MostlyAwesom I put the css rule in main.css as you described on a fresh created meteor project and it seems to show up:

<img width="649" alt="screen shot 2018-05-11 at 10 09 26 am" src="https://user-images.githubusercontent.com/13921582/39937059-6e9098b2-5503-11e8-8443-9d4a7e7782a5.png">

**Comment by msavin** [comment]
Fixed in Meteor Toys 7.0

I tested on todos and a regular starter app.. and would appreciate a further confirmation that its working.

**Comment by kevinkorte** [comment]
@msavin can confirm it appears to be working. Meteor 1.7.0.3 with Collection2 v3, and bringing the allthings package back into this project meteor starts up no problems, and the package seems to be operating normally.

Thank you for fixing this. It's a fantastic collection of tools you've built, and I really appreciate your work and help here.
